src/Avalonia.Controls/Shapes/Arc.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using Avalonia.Media;
+
+namespace Avalonia.Controls.Shapes
+{
+    public class Arc : Shape
+    {
+        /// <summary>
+        /// Defines the <see cref="StartAngle"/> property.
+        /// </summary>
+        public static readonly StyledProperty<double> StartAngleProperty =
+            AvaloniaProperty.Register<Arc, double>(nameof(StartAngle), 0.0);
+
+        /// <summary>
+        /// Defines the <see cref="SweepAngle"/> property.
+        /// </summary>
+        public static readonly StyledProperty<double> SweepAngleProperty =
+            AvaloniaProperty.Register<Arc, double>(nameof(SweepAngle), 0.0);
+
+        static Arc()
+        {
+            StrokeThicknessProperty.OverrideDefaultValue<Arc>(1);
+            AffectsGeometry<Arc>(BoundsProperty, StrokeThicknessProperty, StartAngleProperty, SweepAngleProperty);
+        }
+
+        /// <summary>
+        /// Gets or sets the angle at which the arc starts, in degrees.
+        /// </summary>
+        public double StartAngle
+        {
+            get { return GetValue(StartAngleProperty); }
+            set { SetValue(StartAngleProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the angle at which the arc ends relative to the start angle, in degrees.
+        /// </summary>
+        public double SweepAngle
+        {
+            get { return GetValue(SweepAngleProperty); }
```

Comment:
use `=>` instead of the full fat getter/setter

Reply:
...wait, seriously? I only used `get { return` instead of `get =>` because that's what the rest of Avalonia itself seems to do everywhere I look. I'll make the change, but...am I missing something here?

Reply:
I think all new definitions use the arrow function style. We can't touch old definition just to change the code style.

Reply:
Yea, in this case I do see a lot of code moving to `=>`. This was done in my code base as well and is one of those things that just simplifies things.

> We can't touch old definition just to change the code style.

Um, what!? Refactoring just to refactor, clean-up and improve code is very important with software.

Reply:
> Um, what!? Refactoring just to refactor, clean-up and improve code is very important with software.

Not when you have to do backports/git blame often as we do. See https://brennan.io/2021/06/15/git-less-is-more/ for a discussion.

Reply:
This is very concerning for the long term health of the project. Actively discouraging refactoring should not be done.

Reply:
We're not actively discouraging refactoring, there are just 2 things to consider:

1. Who wants to do it
2. When to do it

So far, no-one has come forward for 1) so we've not discussed 2).

And having mixed syntax for getters and setters is inconsistent, sure, but I don't see how it's "concerning for the health of the project". There are much more important refactors to be done, such as Nullable Reference Types.

Reply:
I can only speak for myself really: because none of the core contributors have brought this up recently we've not got an official line on this.

To give some background: at the start of the project I used StyleCop because I wanted to enforce a strict coding style. This received at _lot_ of pushback from various contributors so I relented and we removed it. As the project grew, I became less concerned about uniform code style and more concerned about getting useful info quickly from `git blame` and being able to backport changes.

Reply:
That's fair. I'll shut up about this topic 😄

Interesting background as well. Personally, I keep code analyzers enabled but disable rules that are too unforgiving. There are inevitably special cases as well. The biggest issue now is build performance as analyzers significantly increase build time.

Reply:
...to clarify, I didn't intend to *challenge* this change request, I was just surprised by it, and had wondered if there may have been some mistake, so I asked for clarification before fulfilling it.

On my own projects, I code by my own rules. Avalonia, however, is not one of my own projects, and so I will gladly play by the rules of the project's core maintainers or whoever else is in charge. _If_ the request pertained to a matter I was more knowledgeable about _and_ I saw a potential drawback of the request, I wouldn't hesitate to bring it up, but the significance of code style is not a matter I am particularly knowledgeable on, so I don't expect that to happen here. Either way though, I respect that ultimately, it's not my call to make.

Reply:
I just wish C# had something like `go fmt` so we could just avoid these discussions ;)

Reply:
@grokys exits equivalent of go fmt and it is https://github.com/dotnet/format

Reply:
Marked as resolved, see https://github.com/AvaloniaUI/Avalonia/pull/6133#discussion_r660799364